vllm/model_executor/input_metadata.py

```diff
@@ -6,11 +6,23 @@
 class InputMetadata:
     """Metadata for input sequences. Used in PagedAttention.
 
+    NOTE: Any python object stored here is not updated when it is
+    cuda-graph replays. If you have values that need to be changed
+    dynamically, it should be stored in tensor. The tensor has to be
+    updated from `CUDAGraphRunner.forward` API.
+
     Args:
         prompt_lens: Lengths of prompts.
-        slot_mapping: The address to write the new KV to of each token.
+        slot_mapping: The index of each token mapped into a physical block
+            in block tables. E.g., if block_size is 32, 35 means it is in
+            the block number 1, 3rd index.
```

Comment:
I polished this a bit. WDYT?
```suggestion
        slot_mapping: The indices of the token slots that input tokens will be stored into.
        E.g., if `slot_mapping` is [35, 2, 17] and the block size is 16, the three tokens
        are stored in the 3rd slot in block 2, 2nd slot in block 0, and 1st slot in block 1,
        respectively. 
```